test/destroy.js

```diff
@@ -33,13 +33,12 @@ tape('listening and destroy twice', function (assert) {
 
 tape('destroy right away', function (assert) {
   const swarm = network()
-
-  swarm.on('close', function () {
```

Comment:
this test was modified because guts (well, nanoresource), won't emit a 'close' event 
if the resource was never opened. Which I think is the right thing to do.

Reply:
Yea makes sense to me also